### PR TITLE
adding a mutationobserver to pfe-content-set

### PR DESCRIPTION
**Pull request by kylebuch8**

### CODE REVIEW

**Comment by kylebuch8** [comment]
@castastrophe In commit https://github.com/patternfly/patternfly-elements/pull/406/commits/dbbc1d3aa84af165b34cdfc7d88803e40ee14eaf I added most of the comments back. I left just a few of the self explanatory comments out.

**Comment by mwcz** [comment]
Somehow prettier was not run on `pfe-content-set.js`.

File: elements/pfe-content-set/src/pfe-content-set.js [hunk 1]
```diff
@@ -130,13 +150,14 @@ class PfeContentSet extends PFElement {
       tabs.setAttribute("pfe-variant", this.variant.value);
```
[the remark this answers is not in the document]
Comment (kylebuch8) [reply elements/pfe-content-set/src/pfe-content-set.js]: @mwcz - This was fixed here: https://github.com/patternfly/patternfly-elements/pull/406/commits/29ed87b52e53d4ce6c87ace781069e990608bff6

File: elements/pfe-content-set/src/pfe-content-set.js [hunk 2]
```diff
@@ -31,94 +31,114 @@ class PfeContentSet extends PFElement {
 
   constructor() {
     super(PfeContentSet, { delayRender: true });
+
+    this._init = this._init.bind(this);
```
Comment (castastrophe) [inline elements/pfe-content-set/src/pfe-content-set.js]: Perhaps this needs to be in pfelement with a setting like `delayRender: true, mutationObserver: true`?

Comment (kylebuch8) [reply elements/pfe-content-set/src/pfe-content-set.js]: I think it's fine where it is for now. MutationObserver is something we should address post 1.0.

**Pull request merged**